README.md

```diff
@@ -1,2 +1,33 @@
-# leaves
-Leaves
+# Leaves
+Leaves application makes it easy to bid and search for items whenever and wherever you are. After you log in, you could either enter as a Bidder or as a Planner.
+
+When the user enters as a Planner, they can :
+* Create an event
+* Manage Events
+* Create Items under each event.
+* View reports of their events
+
+As a Bidder, the user is able to browse, search and bid on items.
+
+### Platform
+
+* Android
+
+## Installation
+
+You'll require two things:
+* The [Java jdk](http://support.apple.com/kb/DL1572?viewlocale=en_US&locale=en_US)
+* [Android Studio](http://developer.android.com/sdk/index.html)
+* Clone the project
+* Start Android Studio and open the project
+
+##Tests
+Several tests were written to test the functionality of this application and they are located in the test directory.
+The tests are based on the MOCKITO unit test.
```

Comment:
One question we can also answer here is: How can I run these tests on my local dev env?

Examples:
for teaspoon in angular(I guess) => $ rake teaspoon
for rspec in ruby on rails => $ bundle exec rspec
for cucumber in ruby => $ rake cucumber

Hope you get the idea.